### ISSUE TITLE
Prevent signatures being available publicly on the web.

### ISSUE THREAD

**Issue by thefeiter** [wp126/Digital-Signature-For-Contact-Form-7#1]
This is an awesome Plugin!

But:
A signature that has been entered is available under 
https://example.com/wp-content/uploads/dscf7_signatures/FIELD_ID-TIMESTAMP.png
to be opened/downloaded by anyone having access to your homepage.
Even without being logged in.

Since the URL follows a quite simple pattern it is pretty easy to retrieve signatures.

This should not be the default case and is preventing me from using this plugin.

**Comment by KZeni** [comment]
Good point. This upload folder should probably be protected via .htaccess (or whatever a particular web server might be using) rules while there's an endpoint provided by this plugin which then retrieves the file (similar to how the Download Monitor plugin defaults to its uploads being blocked from direct access while it goes and serves up the file when accessed via its expected endpoint so it can act on it accordingly [check for sufficient user access to determine if it should even be retrieved or not, etc.])
